tsdb/points_test.go

```diff
@@ -475,6 +475,37 @@ func TestParsePointWithStringWithCommas(t *testing.T) {
 
 }
 
+func TestParsePointEscapedStringsAndCommas(t *testing.T) {
+	// non-escaped comma and quotes
+	test(t, `cpu,host=serverA,region=us-east value="{Hello\"{,}\" World}" 1000000000`,
+		NewPoint(
+			"cpu",
+			Tags{
+				"host":   "serverA",
+				"region": "us-east",
+			},
+			Fields{
+				"value": `{Hello"{,}" World}`,
+			},
+			time.Unix(1, 0)),
+	)
+
+	// escaped comma and quotes
+	test(t, `cpu,host=serverA,region=us-east value="{Hello\"{\,}\" World}" 1000000000`,
+		NewPoint(
+			"cpu",
+			Tags{
+				"host":   "serverA",
+				"region": "us-east",
+			},
+			Fields{
+				"value": `{Hello"{\,}" World}`,
```

Comment:
Only `"` need to be escaped/unescaped for string fields.  Everything else is left as is.  We were escaping/unescaping commas and spaces incorrectly here before as well.